### DISCUSSION TITLE
Clear image folders?

### DISCUSSION THREAD

**Discussion by cuthulino** [ptarmiganlabs/butler-sheet-icons#413] [Q&A] [answered]
Hello

our installation will be quite big with 3 Qlik Sense systems (ervery System 3 contains 3 Nodes).
There will be some thumbnails generated and they will fill up the image dir slowly.

Is it Save to clear the image dir or is it needed for some process in BSI?

**Comment by mountaindude** [comment, marked as answer]
The image dir can be cleared or removed after each BSI execution. 
BSI will create the needed (sub)directories as needed.

More info [here](https://github.com/ptarmiganlabs/butler-sheet-icons#screen-shots-taken-by-butler-sheet-icons).
